src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -16,6 +16,14 @@ const useStyles = makeStyles((theme) => ({
     height: theme.spacing(9),
     margin: "12px auto 8px auto",
   },
+  profileEditButton: {
+    backgroundColor: "transparent",
+    color: theme.color.darkBlue,
+    "&:hover": {
+      backgroundColor: "transparent",
+      color: theme.color.secondaryBlue,
+    },
+  },
```

Comment:
Were these styles already pre-existing in the MUI theme or any of the app level styles?

Reply:
It was already pre-existing in the theme.

Reply:
My question was maybe unclear.

Do we need a specific profileEditButton style object if our theme buttons already cover these styles.

https://material-ui.com/components/buttons/

Just from seeing the source here, it looks like you're trying to achieve the "text button" look in the MUI components.

Reply:
Ah, yeah that styling was redundant. Addressed it in 15ebb3b.